client/src/tutorials/FaucetSeahorse/pages/7.md

```diff
@@ -0,0 +1,13 @@
+# Testing
+
+Now we are ready to build and deploy our program from the 'Build & Deploy' tab. Next navigate to the 'Test' tab and initialize a faucet with the first instruction.
+
+The `mint` will be the token mint account one you created through the command line. Once we have a faucet, we'll need to fill the faucet with tokens which we can do with our `replenish_bitcorn_tokens` instruction.
+
+We used the default setting when minting our token, which was nine decimals. That means to transfer just one token to the faucet we'll need to input one with nine zeros (1000000000) in the `amount` box. We want our faucet to have a lot more than just one token so experiment with adding more zeros. Run the replenish instruction multiple times with different amounts and check the transactions on a block explorer to see if the amounts being transferred match what you expect. You could also try minting more tokens to yourself or the faucet directly through the command line.
+
+```bash
+spl-token mint <TOKEN MINT ACCOUNT> <AMOUNT>
+```
+
+When testing the `dripBitcornTokens` instruction you will need to input the Bitcoin price account. Select 'Pyth' from the drop down menu and scroll to the 'BTC/USD' option.
```

Comment:
Scrolling can be difficult since there are a lot of Pyth accounts in devnet:
```suggestion
When testing the `dripBitcornTokens` instruction you will need to input the Bitcoin price account. Select 'Pyth' from the drop down menu and find the 'BTC/USD' option (you can type it).
```